src/Widgets/BatteryBox.vala

```diff
@@ -24,11 +24,13 @@ public class Power.BatteryBox : Gtk.Grid {
         );
 
         var show_percent_switch = new Gtk.Switch () {
-            halign = END
+            halign = END,
+            valign = Gtk.Align.CENTER,
```

Comment:
```suggestion
            valign = CENTER
```

Let's follow the coding styles around here: implicit namespace of enums and no trailing semicolon for the ending property.